src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessThreadTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Security;
+using System.IO;
+using Xunit;
+
+namespace System.Diagnostics.ProcessTests
+{
+    public class ProcessThreadTests : ProcessTestBase
+    {
+        [Fact]
+        public void TestCommonPriorityAndTimeProperties()
+        {
+            ProcessThreadCollection threadCollection = _process.Threads;
+            Assert.True(threadCollection.Count > 0);
+
+            ProcessThread thread = threadCollection[0];
+
+            if (ThreadState.Terminated != thread.ThreadState)
+            {
+                Assert.Equal(_process.BasePriority, thread.BasePriority);
+                Assert.True(thread.CurrentPriority >= 0, "Thread CurrentPriority");
+                Assert.True(thread.PrivilegedProcessorTime.TotalSeconds >= 0, "Thread PrivilegedProcessorTime");
+                Assert.True(thread.UserProcessorTime.TotalSeconds >= 0, "Thread UserProcessorTime");
+                Assert.True(thread.TotalProcessorTime.TotalSeconds >= 0, "Thread TotalProcessorTime");
+            }
+        }
+
+        [Fact]
+        public void TestStartTimeProperty()
+        {
+            ProcessThreadCollection threadCollection = _process.Threads;
+            Assert.True(threadCollection.Count > 0);
+
+            ProcessThread thread = threadCollection[0];
+
+            if (global::Interop.IsOSX)
+            {
+                Assert.Throws<PlatformNotSupportedException>(() => thread.StartTime);
+                return;
+            }
+
+            if (ThreadState.Initialized != thread.ThreadState)
+            {
+                Assert.True(thread.StartTime.ToUniversalTime() <= DateTime.UtcNow, "Thread StartTime");
```

Comment:
Rather than sleeping, it'd be fine to increase the size of the range by artificially lowering the earlier time by a few seconds.  This test wouldn't be to ensure that we're getting the exact correct value, but rather that the value approximately makes sense, e.g. as it currently stands a thread.StartTime of default(DateTime) would pass the test, but that's obviously wrong.